networkx/algorithms/community/label_propagation.py

```diff
@@ -111,8 +111,8 @@ def label_propagation_communities(G):
 
     Yields
     ------
-    communities : generator
-        Yields sets of the nodes in each community.
+    communities : iterator
+        A dict_values object that yields a set of nodes for each community.
```

Comment:
```suggestion
    communities : iterable
        A dict_values object containing a set of nodes for each community.
```
Just one final cleanup on the docstring. I recommend using a word different than yield so as not to confuse it with `yield`.